Add workaround for old git version for vm (#31)
kyma-project/serverless-manager: module-chart/charts/k3s-tests/README.md

# k3s-tests

## Overview

This project contains a chart with the test Job for the Function Controller.

## Details

> > **CAUTION:** This chart should not be installed with Kyma.

Reviewer: ```suggestion
> **CAUTION:** DO not install this chart with Kyma.
```
